process/configuration/hanoi.json

```diff
@@ -4,7 +4,8 @@
     "geopackagePath": "hanoi_vn_2019_1600m_buffer.gpkg",
     "graphmlName": "hanoi_vn_2019_10000m_pedestrian_osm_20190902.graphml",
     "graphmlProj_name": "hanoi_vn_2019_10000m_pedestrian_osm_20190902_proj.graphml",
-    "folder": "data/input",
```

Comment:
This PR isn't intended to be updating the process folder's config scripts right? It's just for the validation work? If so, please reset any non-validation files and re-push, so they're not in this PR.

Reply:
@thuanguy don't delete the files (hanoi, bern, belfast) then commit the deletion. Just reset the files back to the point at which you branched.

Reply:
I couldn't find the place to reset the file.

Reply:
See this for example: https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

You'll want to return any files you're not actually updating in this PR back to their state at which you branched.